geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaOutputFormat.scala

```diff
@@ -53,11 +53,12 @@ object GeoMesaOutputFormat {
   * Output format that writes simple features using GeoMesaDataStore's FeatureWriterAppend. Can write only
   * specific indices if desired
   */
-class GeoMesaOutputFormat extends OutputFormat[Text, SimpleFeature] {
+class GeoMesaOutputFormat extends OutputFormat[Text, SimpleFeature] with LazyLogging {
 
   override def getRecordWriter(context: TaskAttemptContext): RecordWriter[Text, SimpleFeature] = {
     val params  = GeoMesaConfigurator.getDataStoreOutParams(context.getConfiguration)
     val indices = GeoMesaConfigurator.getIndicesOut(context.getConfiguration)
+    logger.info("new gm record writer")
```

Comment:
can probably remove this unless it's still useful, in which case should make it a little more official. can also switch to `StrictLogging`, since the logger will always be used (not just in e.g. case of exception being thrown)